docs/framework/data/adonet/ef/mathematical-functions.md

```diff
@@ -8,31 +8,31 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides math functi
-|`ATN2(` `expression`, `expression``)`|Returns the angle, in radians, whose tangent is between the two specified numeric expressions.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ATN2(9, 8)`|  
-|`CEILING(` `expression` `)`|Converts the specified expression to the smallest integer that is greater than or equal to it.<br /><br /> **Arguments**<br /><br /> `expression`: An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Return Value**<br /><br /> An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#SQLSERVER_CEILING](../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitysql.cs#sqlserver_ceiling)]
+|`ABS(expression)`|Performs the absolute value function.<br /><br /> **Arguments**<br /><br /> `expression`: An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Return Value**<br /><br /> The absolute value of the specified expression.<br /><br /> **Example**<br /><br /> `SqlServer.ABS(-2)`|  
+|`ACOS(expression)`|Returns the arccosine value of the specified expression.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ACOS(.9)`|  
+|`ASIN(expression)`|Returns the arcsine value of the specified expression.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ASIN(.9)`|  
+|`ATAN(expression)`|Returns the arctangent value of the specified numeric expression.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ATAN(9)`|  
+|`ATN2(expression, expression)`|Returns the angle, in radians, whose tangent is between the two specified numeric expressions.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ATN2(9, 8)`|  
+|`CEILING(expression)`|Converts the specified expression to the smallest integer that is greater than or equal to it.<br /><br /> **Arguments**<br /><br /> `expression`: An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Return Value**<br /><br /> An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#SQLSERVER_CEILING](../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitysql.cs#sqlserver_ceiling)]
  [!code-sql[DP EntityServices Concepts#SQLSERVER_CEILING](../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#sqlserver_ceiling)]|  
```

Comment:
This belongs in the second column of the previous (Ceiling) row. See https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/mathematical-functions?branch=pr-en-us-6494.